libdnf/rpm/solv/solv_map.hpp

```diff
@@ -154,6 +157,12 @@ inline SolvMap::SolvMap(const Map * other) {
 
 inline SolvMap::SolvMap(const SolvMap & other) : SolvMap(other.get_map()) {}
 
+inline SolvMap::SolvMap(SolvMap && other) {
+    map.size = other.map.size;
+    map.map = other.map.map;
+    other.map.size = 0;
+    other.map.map = nullptr;
```

Comment:
OK. Add move constructor is good idea. We must be sure to don't use moved out object with "unsafe" methods (eg. ad_unsafe). But it is not a new problem.

Reply:
And. Please, make it `noexcept`.

Reply:
@jrohel fixed